Description: Make hard install friendly
Added ability to flag multiple devices to be ignored, and added more house cleaning to make friendly with hard installed OS.
 .
 tbs (0.1-1) unstable; urgency=medium
 .
   * Initial release (Closes: #nnnn)  <nnnn is the bug number of your ITP>
Author: Colby Connolly <protheophage@gmail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: 2017-09-06

--- tbs-0.1.orig/tbs
+++ tbs-0.1/tbs
@@ -41,13 +41,22 @@ echo " "
 
 #BLACK (1)
 
-#Find live usb
+#Find devices to be ignored
 find /media/root -maxdepth 2 -iname 'iignoree_meee' > /root/Documents/live.txt
 
-#Identify live usb
-findmnt -M $(grep -o '/media/root/device..' /root/Documents/live.txt) -o source > /root/Documents/flagd.txt
+#Create Counter
+l=$(grep -c '.' /root/Documents/live.txt)
+m=1
+
+#Identify devices to be ignored
+touch /root/Documents/flagd.txt
+while [ $m -le $l ]
+do
+findmnt -M $(head -$m /root/Documents/live.txt | tail -1 | grep -o '/media/root/device..') -o source >> /root/Documents/flagd.txt
+m=`expr $m + 1`
+done
 
-#Remove live usb from imaging list
+#Remove ignored devices from imaging list
 grep -vwE "$(grep -o "/dev/sd." /root/Documents/flagd.txt)" /root/Documents/devs.txt > /root/Documents/fdevs.txt
 
 #Clean up imaging list
@@ -71,12 +80,15 @@ done
 
 #HOUSE CLEANING (1)
 
-#Remove text docs created by tbs
+#Remove text docs and  mount directories created by tbs
+	#to be more friendly with hard installed Linux
 rm -f /root/Documents/mnt.txt
 rm -f /root/Documents/devs.txt
 rm -f /root/Documents/live.txt
 rm -f /root/Documents/flagd.txt
 rm -f /root/Documents/fdevs.txt
 rm -f /root/Documents/ilist.txt
+umount -a
+rmdir /media/root/dev*
 
 #Written by Colby Connolly 2016
